backends/tfhe-cuda-backend/cuda/include/helper.h

```diff
@@ -1,10 +1,16 @@
 #ifndef HELPER_H
 #define HELPER_H
 
+extern bool p2p_enabled;
+
 extern "C" {
 int cuda_setup_multi_gpu();
 }
 
-void multi_gpu_checks(uint32_t gpu_count);
+int get_active_gpu_count(int num_inputs, int gpu_count);
+
+int get_num_inputs_on_gpu(int total_num_inputs, int gpu_index, int gpu_count);
+
+int get_gpu_offset(int total_num_inputs, int gpu_index, int gpu_count);
```

Comment:
I will rename the helper.h file :+1: 

Regarding your question about the number of threads used for testing, actually I have to restrict it otherwise we run out of memory on the GPU. Since David is working on another PR to change the CI and there he had to restrict it for the CPU I'm using the same value here as will be used later after his changes.